src/themes/playful.css

```
@@ -296,16 +312,16 @@
   /**
    * Transitions
    */
-  --wa-transition-normal: 200ms;
-  --wa-transition-fast: 100ms;
+  --wa-transition-normal: 250ms;
+  --wa-transition-fast: 150ms;
   --wa-transition-faster: 50ms;
 
   /**
    * Component groups
    */
-  --wa-form-controls-background: var(--wa-color-surface-default);
+  --wa-form-controls-background: transparent;
 
-  --wa-form-controls-corners: var(--wa-corners-s);
+  --wa-form-controls-corners: var(--wa-corners-m);
```

```suggestion
  --wa-form-controls-corners: var(--wa-corners-pill);
```

We'll have better luck ensuring that form controls are always pill shaped by using this utility property.